### DISCUSSION TITLE
How to prevent namespace from being deleted?

### DISCUSSION THREAD

**Discussion by eisig** [fluxcd/flux2#1289] [Q&A] [answered]
We created some other resources in the namespace not managed by flux. If I remove the flux Kustomization from git. It would delete the entire namespace .

**Comment by stefanprodan** [comment, marked as answer]
From the docs https://toolkit.fluxcd.io/components/kustomize/kustomization/#garbage-collection

> You can disable pruning for certain resources by either labeling or annotating them with:
> `kustomize.toolkit.fluxcd.io/prune: disabled`

**Reply by eisig** [reply]
Thanks
